.travis.yml

```diff
@@ -0,0 +1,51 @@
+# Travis Continuous Integration Configuration File, based on ..
+language:
+  - cpp
+  - python
+python:
+  - "2.7"
+compiler:
+  - gcc
+env:
+  - OPENCV_VERSION=2 ROS_DISTRO=hydro
+  - OPENCV_VERSION=3 ROS_DISTRO=hydro
```

Comment:
I just want to test both opencv2 and opencv3, unfortunately travis is hosted on ubuntu precise so we could not test on indigo, so this is pseudo testing on hydro and changing libopencv-dev to opencv{2,3} at  https://github.com/k-okada/vision_opencv/blob/add_travis/.travis.yml#L35